wasm/lib/src/vm_class.rs

```diff
@@ -0,0 +1,78 @@
+use js_sys::TypeError;
+use rustpython_vm::VirtualMachine;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use wasm_bindgen::prelude::*;
+
+// It's fine that it's thread local, since WASM doesn't even have threads yet
+thread_local! {
```

Comment:
I'm very curious how this translates to in WASM? Do you have any demo or link to rust-wasm documentation?

Reply:
I don't quite know, I'd assume there's a sort of thread constructor that the Rust runtime runs that includes instantiating thread local variables, and since WASM doesn't have threads it only runs it once. There's also [this post I found](https://rustwasm.github.io/2018/10/24/multithreading-rust-and-wasm.html#atomic-instructions) explaining how `Atomic*` numbers are compiled down to normal variables for WASM, so I'd assume something similar for `thread_local!`.

Reply:
No worries, as long as it runs we are good to go.